src/OpenTelemetry/Metrics/AggregatorStore.cs

```diff
@@ -105,27 +107,34 @@ internal IAggregator[] FindMetricAggregators(ReadOnlySpan<KeyValuePair<string, o
                 Array.Sort<string, object>(tagKey, tagValue);
             }
 
+            string name = this.instrument.Name;
```

Comment:
the instrument name (and in future view name as well), will be part of the Aggregator anyway. and we have the ability to map an incoming measurement to multiple aggregators. So why do we have to add the viewname/instrumentname to the key?

Reply:
An instrument may contribute to multiple views.

Reply:
Also, I do not agree that an Instrument is part of the Aggregator. An Aggregator is just an aggregator without regards to which instrument contributes values to it.

However, an Aggregator (when we collect) does need to map it's metric names (usually based on instrument info and/or view info). But, it does not HAVE an instrument.

Reply:
> I do not agree that an Instrument is part of the Aggregator

Nobody said otherwise.